source/blog/2015-06-12-ember-1-13-0-released.md

```diff
@@ -0,0 +1,578 @@
+---
+title: Ember.js 1.13.0 and 2.0 Beta Released
+author: Matthew Beale
+tags: Releases
+---
+
+We are pleased to announce the release of both Ember.js 1.13.0 and the
+first beta in the 2.0 series. This comes as the thirteenth cycle of our
+release process that began just after 1.0 was released.
+
+The 1.13 release represents the effort of at least 43 contributors
+across over 680 commits.
+
+## New Features in Ember.js 1.13
+
+The Glimmer rendering engine, [announced at EmberConf](https://www.youtube.com/watch?v=o12-90Dm-Qs&feature=youtu.be&t=47m21s) and introduced to Ember.js
+Canary on [May 5th](/blog/2015/05/05/glimmer-merging.html), is released today
+in Ember.js 1.13. Glimmer is the third overhaul of Ember's 1.x rendering
+layer (wow, right?), and dramatically improves rerender performance in many
+common scenarios. Additionally it lays important groundwork for the Ember 2.x
+development model.
+
+**We are tremendously excited to bring Glimmer to existing applications as
+a 1.x compatible minor release.** The herculean efforts of the Ember core team,
+addon authors, and community to achieve this release has been inspiring.
+Thank you! You are too many to possibly name.
+
+**1.13 also marks the final minor release of the Ember 1.x cycle.** As such,
+it contains a number of deprecations ahead of Ember 2.0. Resolving these
+deprecations in application code aligns that application with Ember's public
+API in 2.0. Any Ember app running 1.13 without causing a deprecation notice
+to fire should upgrade to 2.0 without any changes.
+
+#### Improved Rerender Performance
+
+Previous iterations of Ember's rendering layer were based on granular
+observation. Where a piece of dynamic content was rendered (such as a
+new array, or new component) the old DOM and possibly-invalid child views
+were discarded and new DOM and views created from scratch. Additionally,
+granular observation was computationally expensive to initialize and maintain
```

Comment:
this seems incorrect, the granular observation hasn't changed. Rather we are able to maintain stability inlight of common roots (in an each) changing.